lib/hooks/useDateFormatter.ts

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from 'react';
+import useSiteData from './useSiteData';
+
+/**
+ * Gets forecasted and solar panel data for a single site
```

Comment:
great having this comment here, but this line is inaccurate (copied/pasted but not changed perhaps?) and should be fixed